src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,68 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(name = "movie_id")
+    private Movie movie;
+    @OneToOne

```

Comment:
why 1-1 ?

Reply:
at this moment one cinemaHall can have only one MovieSession. During the day it can have many.
As I understand when we talk about many smth to smth, means that at this moment it can have many. This is the same as a car has an engine at this moment and in 2 hours it has a new engine and will ManyToOne, but in this case, we always say OneToOne (car and engine, the car can have only one engine at this moment)
Am I not right?